managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerFactoryImpl.java

```diff
@@ -408,7 +408,7 @@ public void initializeFailed(ManagedLedgerException e) {
                         pendingLedger.ledger.asyncClose(new CloseCallback() {
                             @Override
                             public void closeComplete(Object ctx) {
-                                // no-op
+                                log.info("[{}] Closed pending initializer managed ledger.", name);
```

Comment:
probably we want to leave this at `debug` level

Reply:
good catch, I'll fix it.